src/injected/visualization/table-headers-formatter.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { DialogRenderer } from '../dialog-renderer';
+import { AssessmentVisualizationInstance } from '../frameCommunicators/html-element-axe-results-helper';
+import { FailureInstanceFormatter } from './failure-instance-formatter';
+import { DrawerConfiguration } from './formatter';
+
+export interface HeadingStyleConfiguration {
```

Comment:
`HeadingStyleConfiguration` and `StyleComputer` don't appear to be used; are these copy-paste leftovers?